docs/python/testing.md

```diff
@@ -313,6 +313,21 @@ Below are all the supported commands for testing with the Python extension in VS
 |  **Testing: Focus on Test Explorer View** | Open the Test Explorer view. Similar to **Testing: Focus on Python View** on versions prior to 2021.9.
 |  **Test: Stop Refreshing Tests** | Cancel test discovery. |
 
+## IntelliSense for pytest fixtures and parameterization
```

Comment:
This H2 text will likely result in word wrap in the right navigation "In this topic" list.
Could it be shortened to just "IntelliSense for pytest"?